docs/cs-proof.md

```diff
@@ -752,10 +874,21 @@ The verifier flattens constraints:
 \\]
 where each of \\(\mathbf{w}\_L, \mathbf{w}\_R, \mathbf{w}\_O\\) has length \\(n\\) and \\(\mathbf{w}\_V\\) has length \\(m\\).
 
+The verifier [pads](#padding-mathbflx-and-mathbfrx-for-the-inner-product-proof) the generators \\(\mathbf{G},\mathbf{H}\\) and challenges \\(\mathbf{y}^n\\) to a power-of-two \\(n^{+}\\):
```

Comment:
This also talks about the generator-extension as padding, but I'm not sure if it's exactly the right metaphor: when we do proving and verification, the generators are already given up front with some capacity, and we use the first power-of-two subset of them.  So rather than padding the generators, it's selecting a bigger subslice of a longer set of generators.